test/UnitTests/TestFramework.UnitTests/Exceptions/BinaryFormatterExceptionSerializationTests.cs

```diff
@@ -35,7 +35,7 @@ private void VerifySerialization(Action actionThatThrows)
             var mem = new MemoryStream();
             var formatter = new BinaryFormatter
             {
-                Binder = new FormatterBinder(),
+                Binder = new FormatterBinder(ex.GetType()),
```

Comment:
Nice! That's a better idea than my suggestion to add `Type` as a parameter of `VerifySerialization`. Thanks!!

Reply:
Let's hope it makes the analyzer happy finally :D